cdap-cli/src/main/java/co/cask/cdap/cli/command/app/ListAppsCommand.java

```diff
@@ -77,7 +77,7 @@ public String getPattern() {
 
   @Override
   public String getDescription() {
-    return String.format("Lists all %s, optionally filtered by artifact name and version.",
+    return String.format("Lists all %s, optionally filtered by artifact name and version",
```

Comment:
In general, our style for tables is that single sentences are treated as phrases and do not end with a period. Multi-sentence paragraphs have trailing periods.